core/framework/cloudhelpers/deployments.py

```diff
@@ -12,13 +12,36 @@
 import yaml
 
 
-def _read_render_config(file_name, template_args={}):
-    with open(file_name) as f:
-        content = f.read()
-    if not template_args == {}:
-        return jinja2.Template(content).render(**template_args)
+def _read_render_config(file_name, template_args={}, load_path=[]):
+    '''Use load_path to set jinja env loader, if there are blocks in level yaml.
+    
+    Parameters:
+        file_name (str): Relative file path with name. example core/levels/leastprivilege/c1project/c1project.yaml.
+            if load_path provided, should be file name only. example c1project.yaml
+        template_args (dict, optional): Dictionary of arguments to use when rendering the top level configuration template using Jinja2.
+            Keys should be strings that correspond to the names of variables in the Jinja template, and each corresponding value should be the passed value of the variable.
+            If not supplied, the top level configuration will not be treated as a template.
+        load_path (str): Relative path example core/levels/leastprivilege/c1project/
+    '''
+
+    if not load_path == []:
```

Comment:
Tested with a couple thunder levels. Should be safe for deployment of all levels. Possible to remove else block in the future